Purpose: posix_fadvise() is not present in Debian/kfreebsd. Provide
         compatibility define for platforms which do not support it
	 and make it a no-op.
Fixes:   #534337
Authour: Petr Salinger <Petr.Salinger@seznam.cz>
Status:  Submitted upstream.
---
--- a/makeboot.c
+++ b/makeboot.c
@@ -17,6 +17,9 @@
 # include <sys/types.h>
 # include <sys/stat.h>
 # include <fcntl.h>
+#ifndef POSIX_FADV_SEQUENTIAL
+#define posix_fadvise(fd, off, len, adv)	(-1)
+#endif
 static int o_flags = O_RDONLY;
 #endif
 
